packages/components/src/visually-hidden/hook.js

```diff
@@ -1,26 +0,0 @@
-/**
- * External dependencies
- */
-import { cx } from '@emotion/css';
-
-/**
- * Internal dependencies
- */
-import * as styles from './styles';
-
-/**
- * @param {import('../ui/context').PolymorphicComponentProps<{ children: import('react').ReactNode }, 'div'>} props
- */
-export function useVisuallyHidden( { className, ...props } ) {
-	// circumvent the context system and write the classnames ourselves
-	const classes = cx(
-		'components-visually-hidden',
```

Comment:
I don't understand how the `components-visually-hidden` class is still added in snapshot tests, since it doesn't look like it's explicitly added to the component anymore

Reply:
Ok, it looks like it is added [when calling `useContextSystem`](https://github.com/WordPress/gutenberg/blob/b6a8a288852019f5a809cc5f3ad1c268905f739d/packages/components/src/ui/context/use-context-system.js#L59)